testsuite/libra-fuzzer/tests/coverage.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_fuzzer::FuzzTarget;
+use std::{env, fs, path::PathBuf};
+
+#[test]
+#[ignore]
```

Comment:
I think this is generally good to do. Can you add a comment here explaining all the discussions we've had? specifically:
* why this is a test and not a binary
* what tools this is expected to be used with
* why this is marked `#[ignore]`

Also, cc @sausagee re coverage work.

Reply:
actually, it looks like `#[ignore]` doesn't work. Not sure why but I cannot get cargo to accept the `--ignored` flag, so I have a less ideal fix: return early if the directory does not exist.